build_tools/azure/combine_coverage_reports.sh

```diff
@@ -8,11 +8,11 @@ source build_tools/shared.sh
 activate_environment
 
 # Combine all coverage files generated by subprocesses workers such
-# such as pytest-xdist and joblib/loky:
+# as pytest-xdist and joblib/loky:
 pushd $TEST_DIR
 coverage combine --append
 coverage xml
 popd
 
 # Copy the combined coverage file to the root of the repository:
-cp $TEST_DIR/coverage.xml $BUILD_REPOSITORY_LOCALPATH
+cp $TEST_DIR/coverage.xml .
```

Comment:
Comment for potential other reviewers: `BUILD_REPOSITORY_LOCALPATH` is a Azure-specific environment variable that points where the scikit-learn root folder is located.

Maybe GitHub has an equivalent environment variable, but it feels slightly simpler to rely on the assumption that the current working directory is the same as where the repo root folder.